packages/remediations/src/NewRemediationWizardHelper.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component, Fragment } from 'react';
+import RemediationWizard from './NewRemediationWizard';
+
+class NewRemediationWizard extends Component {
```

Comment:
What is is the reason behind this component? And why is it a class component? :thinking:

Reply:
It's there to keep the original API and the OpenWizard method. It is rather a temporary solution.